server/pull_request.go

```diff
@@ -413,3 +422,16 @@ func (s *Server) CleanOutdatedPRs() {
 	}
 	mlog.Info("Finished update the outdated prs in the mattermod database....")
 }
+
+func (s *Server) isBlockPRMerge(label string) bool {
+	return label == s.Config.BlockPRMergeLabel
```

Comment:
Maybe we check if `s.Config.BlockPRMergeLabel` is a prefix of `label`. This way we can set `s.Config.BlockPRMergeLabel` to `Do Not Merge` and don't have to configure the individual labels.